selinux.go

```diff
@@ -3,10 +3,13 @@
 package buildah
 
 import (
+	selinux "github.com/opencontainers/go-selinux"
 	"github.com/opencontainers/runtime-tools/generate"
 )
 
 func setupSelinux(g *generate.Generator, processLabel, mountLabel string) {
-	g.SetProcessSelinuxLabel(processLabel)
-	g.SetLinuxMountLabel(mountLabel)
+	if processLabel != "" && selinux.EnforceMode() != selinux.Disabled {
```

Comment:
There is an interface selinux.GetEnabled() should be enough, but I don't think you even need this.  The processLabel will be "" if SELinux is enabled.